fe/fe-core/src/main/java/com/starrocks/staros/StarMgrIdGenerator.java

```diff
@@ -0,0 +1,20 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.staros;
+
+import com.staros.util.IdGenerator;
+import com.starrocks.catalog.CatalogIdGenerator;
+
+// wrapper for star mgr to use catalog id generator
+public class StarMgrIdGenerator extends IdGenerator {
```

Comment:
Make sure you only use this on the master node. Because it will write bdb log.

Reply:
yes, leader will be checked before starmgr accepts write request.